sagemaker-geospatial/vector-enrichment-reverse-geocoding/vector-enrichment-reverse-geocoding.ipynb

```diff
@@ -0,0 +1,902 @@
+{
```

Comment:
<p>This data is from a third-party source that we are not in control of - as such, the dataset can be rendered inaccessible or moved at any time, breaking the notebook. Please use s3://sagemaker-sample-files/datasets/tabular/california_housing/cal_housing.tgz instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3859/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Thanks, I changed the code to incorporate the dataset from sagemaker-sample-files instead.